### ISSUE TITLE
Panic on Users -> LDAP tab

### ISSUE THREAD

**Issue by zerkms** [Caiyeon/goldfish#175]
```
{"time":"2017-10-18T02:30:30.035780232Z","level":"-","prefix":"echo","file":"asm_amd64.s","line":"510","message":"[\x1b[31mPANIC RECOVER\x1b[0m] runtime error: invalid memory address or nil pointer dereference goroutine 86 [running]:\ngithub.com/caiyeon/goldfish/vendor/github.com/labstack/echo/middleware.RecoverWithConfig.func1.1.1(0x2322f60, 0x1000, 0xc4207d0000, 0x2f87bc0, 0xc4204ba850)\n\tF:/gopath/src/github.com/caiyeon/goldfish/vendor/github.com/labstack/echo/middleware/recover.go:75 +0x12a\npanic(0x180ea00, 0x2fcf440)\n\tC:/Go/src/runtime/panic.go:491 +0x283\ngithub.com/caiyeon/goldfish/vault.AuthInfo.ListLDAPUsers(0x1ac90bd, 0x5, 0xc420019350, 0x24, 0x0, 0x0, 0x10, 0x170b560, 0xc42000f601, 0xc420453b10, ...)\n\tF:/gopath/src/github.com/caiyeon/goldfish/vault/ldap.go:73 +0xb3\ngithub.com/caiyeon/goldfish/handlers.GetLDAPUsers.func1(0x2f87bc0, 0xc4204ba850, 0x0, 0x0)\n\tF:/gopath/src/github.com/caiyeon/goldfish/handlers/ldap.go:40 +0xb8\ngithub.com/caiyeon/goldfish/vendor/github.com/labstack/echo.(*Echo).add.func1(0x2f87bc0, 0xc4204ba850, 0xd, 0x1b02faa)\n\tF:/gopath/src/github.com/caiyeon/goldfish/vendor/github.com/labstack/echo/echo.go:467 +0x87\nmain.main.func1.1(0x2f87bc0, 0xc4204ba850, 0xc4204d6b10, 0x2f79a00)\n\tF:/gopath/src/github.com/caiyeon/goldfish/server.go:135 +0xa4\ngithub.com/caiyeon/goldfish/vendor/github.com/labstack/echo/middleware.GzipWithConfig.func1.1(0x2f87bc0, 0xc4204ba850, 0x0, 0x0)\n\tF:/gopath/src/github.com/caiyeon/goldfish/vendor/github.com/labstack/echo/middleware/compress.go:92 +0x17f\ngithub.com/caiyeon/goldfish/vendor/github.com/labstack/echo/middleware.BodyLimitWithConfig.func1.1(0x2f87bc0, 0xc4204ba850, 0x0, 0x0)\n\tF:/gopath/src/github.com/caiyeon/goldfish/vendor/github.com/labstack/echo/middleware/body_limit.go:87 +0x19f\ngithub.com/caiyeon/goldfish/vendor/github.com/labstack/echo/middleware.RecoverWithConfig.func1.1(0x2f87bc0, 0xc4204ba850, 0x0, 0x0)\n\tF:/gopath/src/github.com/caiyeon/goldfish/vendor/github.com/labstack/echo/middleware/recover.go:82 +0xe1\ngithub.com/caiyeon/goldfish/vendor/github.com/labstack/echo/middleware.LoggerWithConfig.func2.1(0x2f87bc0, 0xc4204ba850, 0x0, 0x0)\n\tF:/gopath/src/github.com/caiyeon/goldfish/vendor/github.com/labstack/echo/middleware/logger.go:111 +0x121\ngithub.com/caiyeon/goldfish/vendor/github.com/labstack/echo.(*Echo).ServeHTTP.func1(0x2f87bc0, 0xc4204ba850, 0xc4200541f8, 0x1aa8da0)\n\tF:/gopath/src/github.com/caiyeon/goldfish/vendor/github.com/labstack/echo/echo.go:558 +0x108\ngithub.com/caiyeon/goldfish/vendor/github.com/labstack/echo.(*Echo).ServeHTTP(0xc4200541a0, 0x2f79a00, 0xc4203981c0, 0xc4201aec00)\n\tF:/gopath/src/github.com/caiyeon/goldfish/vendor/github.com/labstack/echo/echo.go:567 +0x228\nnet/http.serverHandler.ServeHTTP(0xc42005edd0, 0x2f79a00, 0xc4203981c0, 0xc4201aec00)\n\tC:/Go/src/net/http/server.go:2619 +0xb4\nnet/http.(*conn).serve(0xc4200620a0, 0x2f7ad00, 0xc420530000)\n\tC:/Go/src/net/http/server.go:1801 +0x71d\ncreated by net/http.(*Server).Serve\n\tC:/Go/src/net/http/server.go:2720 +0x288\n\ngoroutine 1 [IO wait]:\ninternal/poll.runtime_pollWait(0x7f5ae9ec7fe8, 0x72, 0xffffffffffffffff)\n\tC:/Go/src/runtime/netpoll.go:173 +0x57\ninternal/poll.(*pollDesc).wait(0xc4200e8198, 0x72, 0xc420473900, 0x0, 0x0)\n\tC:/Go/src/internal/poll/fd_poll_runtime.go:85 +0xae\ninternal/poll.(*pollDesc).waitRead(0xc4200e8198, 0xffffffffffffff00, 0x0, 0x0)\n\tC:/Go/src/internal/poll/fd_poll_runtime.go:90 +0x3d\ninternal/poll.(*FD).Accept(0xc4200e8180, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)\n\tC:/Go/src/internal/poll/fd_unix.go:334 +0x1e2\nnet.(*netFD).accept(0xc4200e8180, 0x40c6ca, 0xc420473ad8, 0x429b66)\n\tC:/Go/src/net/fd_unix.go:238 +0x42\nnet.(*TCPListener).accept(0xc42000c5e0, 0xc420000180, 0x23256a8, 0xc420473af0)\n\tC:/Go/src/net/tcpsock_posix.go:136 +0x2e\nnet.(*TCPListener).AcceptTCP(0xc42000c5e0, 0x401137, 0xc420063700, 0xc420473b50)\n\tC:/Go/src/net/tcpsock.go:234 +0x49\ngithub.com/caiyeon/goldfish/vendor/github.com/labstack/echo.tcpKeepAliveListener.Accept(0xc42000c5e0, 0xc420473b68, 0xc420473b70, 0xc420473b60, 0x6961b8)\n\tF:/gopath/src/github.com/caiyeon/goldfish/vendor/github.com/labstack/echo/echo.go:701 +0x2f\ngithub.com/caiyeon/goldfish/vendor/github.com/labstack/echo.(*tcpKeepAliveListener).Accept(0xc42000c5e8, 0x2324f80, 0xc420063680, 0x2f7adc0, 0xc4203f2870)\n\t<autogenerated>:1 +0x3c\nnet/http.(*\n"}
```
goldfish: 0.8.3

I'm running vault with ldap and I only have ldap group -> policy mapping, I don't have ldap users mapped specifically (it looks like it fails when lists it)

**Comment by Caiyeon** [comment]
This has been fixed in #169 and will be in `v0.7.4` when it is released

**Comment by zerkms** [comment]
Right, I see, sorry.
